pkgs/applications/misc/sblg/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl, expat }:
+
+stdenv.mkDerivation rec {
+  pname = "sblg";
+  version = "0.5.11";
+
+  outputs = [ "out" "dev" "man" ];
+
+  src = fetchurl {
+    url = "https://kristaps.bsd.lv/sblg/snapshots/sblg.tar.gz";
+    sha512 = "2741ce27172d702b2f0165d2fe796896fbc2a08b838e486aa61b9ccfe629ba0a09d5e1e9bbea168425c592bb39ae1cc073e0259772009bad5c7186f64558c93c";
+  };
+
+  buildInputs = [ expat ];
+
+  configurePhase = ''
+    ./configure PREFIX=''${!outputDev} \
+                BINDIR=''${!outputBin}/bin \
+                MANDIR=''${!outputMan}/share/man
+  '';
```

```suggestion
  configurePhase = ''
    runHook preConfigure
    ./configure PREFIX=''${!outputDev} \
                BINDIR=''${!outputBin}/bin \
                MANDIR=''${!outputMan}/share/man
    runHook postConfigure
  '';
```

when overriding a phase, [you must call its hooks](https://github.com/NixOS/nixpkgs/blob/f8131a5a2c4e49b599cfc8a0a114dd5905ae21a0/doc/stdenv/stdenv.chapter.md#phases-sec-stdenv-phases)